src/app/cli/src/coda.ml

```diff
@@ -421,23 +421,21 @@ integration_tests]
 
 let coda_commands log =
   let group =
-    let module Coda_peers_test = Coda_peers_test.Make (Kernel) in
-    let module Coda_block_production_test =
-      Coda_block_production_test.Make (Kernel) in
-    let module Coda_shared_prefix_test = Coda_shared_prefix_test.Make (Kernel) in
-    let module Coda_restart_node_test = Coda_restart_node_test.Make (Kernel) in
-    let module Coda_shared_state_test = Coda_shared_state_test.Make (Kernel) in
-    let module Coda_receipt_chain_test = Coda_receipt_chain_test.Make (Kernel) in
-    let module Coda_transitive_peers_test =
-      Coda_transitive_peers_test.Make (Kernel) in
+    let module Coda_peers_test = Coda_peers_test in
```

Comment:
There's no need to alias these anymore. This is equivalent to saying `let x = x in x`.